<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>615</y>
      <width>851</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>16</width>
      <height>571</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="run">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>240</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_calib">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load calibration</string>
    </property>
   </widget>
   <widget class="dragZoomQLabel" name="result_final">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="find_kb">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>200</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Kilobots</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>280</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>3,2,1,0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>280</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cam order:</string>
    </property>
   </widget>
   <widget class="QSlider" name="cannyThresh_slider">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>120</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>15</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="cannyThresh_label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>120</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
   <widget class="QLabel" name="houghAcc_label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>140</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>57</string>
    </property>
   </widget>
   <widget class="QSlider" name="houghAcc_slider">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>140</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>57</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="kbMin_label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>160</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QSlider" name="kbMin_slider">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>160</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
    <property name="value">
     <number>7</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="kbMax_label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>180</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>17</string>
    </property>
   </widget>
   <widget class="QSlider" name="kbMax_slider">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>180</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
    <property name="value">
     <number>17</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>885</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>dragZoomQLabel</class>
   <extends>QLabel</extends>
   <header>dragzoomqlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kbMax_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>kbMax_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>855</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kbMin_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>kbMin_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>855</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cannyThresh_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cannyThresh_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>855</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>houghAcc_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>houghAcc_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>855</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
